mapie/utils.py

```diff
@@ -398,9 +406,70 @@ def check_nan_in_aposteriori_prediction(X: ArrayLike) -> None:
     WARNING: at least one point of training set belongs to every resamplings.
     Increase the number of resamplings
     """
-    if np.any(np.all(np.isnan(X), axis=1), axis=0):
+    if len(X.shape) == 1:
```

Comment:
Is it possible to have such a shape ? In practice, base sklearn estimators should complain and you would need to reshape `X` with `.reshape(-1, 1)`.